src/file.ts

```diff
@@ -1740,12 +1745,20 @@ class File extends ServiceObject<File> {
       md5,
     });
 
+    const progressStream = new ProgressStream();
+    progressStream.on('progress', bytesRead => {
```

Comment:
We should just forward this event to the returned stream, rather than the handler (IMO). Keeps with the streaming API, so we don't have to mix callbacks/streams.